libs/core-ui/src/lib/Interfaces/IDataset.ts

```diff
@@ -19,6 +19,8 @@ export interface IDataset {
   features: unknown[][];
   feature_names: string[];
   categorical_features: string[];
+  is_large_data_scenario?: boolean;
```

Comment:
I think you need to whitelist the names to make the linter pass here:

https://github.com/microsoft/responsible-ai-toolbox/blob/2bd09f98fdffe3e18c6a04f92b127e727da747c9/.eslintrc/.eslintrc.custom.eslintrc

Reply:
Done. Thanks for pointing this out.